### PR TITLE
upgrade to Node 8 (respec now uses async/await)

### PR DESCRIPTION
**Pull request by vivienlacourba**
requested by @dontcallmedom

### CODE REVIEW

**Comment by alvestrand** [comment]
Needs a respec upgrade?
(we're not auto-upgrading respec due to past issues)

**Comment by vivienlacourba** [comment]
`webrtc-stats.html` has `<script class="remove" src="https://www.w3.org/Tools/respec/respec-w3c-common">` so it uses the latest published respec version (currently 1.7.1).

@dontcallmedom noticed recent travis builds were failing because respec now uses `async` and Travis was still running Node 6.

This was the error he got from https://travis-ci.org/w3c/webrtc-stats/builds/290375085:
```
node support/respec/tools/respec2html.js -e --src file://`pwd`/webrtc-stats.html --out build/output.html
/home/travis/build/w3c/webrtc-stats/support/respec/tools/respecDocWriter.js:31
async function writeTo(outPath, data) {
^^^^^^^^
SyntaxError: Unexpected token function
```

Switching to Node 8 fixes this particular issue but repesc still fails https://travis-ci.org/w3c/webrtc-stats/builds/290456592:
```
node support/respec/tools/respec2html.js -e --src file://`pwd`/webrtc-stats.html --out build/output.html
😱 ReSpec error: [object Object]
make: *** [build/output.html] Error 1
```

Good or bad news we are not the only one having issues
https://github.com/w3c/spec-generator/pull/44#issuecomment-338206307

@marcoscaceres, Dom noticed a `ReferenceError: respecConfig is not defined` error in the console.

**Comment by vivienlacourba** [comment]
@dontcallmedom found teh cause of this issue @tripu is filling a respec issue for this bug
```
<dom> I get a console error when hitting "save as" in the respec menu
<dom> ^tripu vivien
<dom> I'm disappearing for good now; if not fixing the bug, could one of you raise a respec issue with this?
<dom> Uncaught ReferenceError: js_source_text is not defined
<dom>     at new e.exports.Beautifier (require.js:1)
<dom>     at e.exports (require.js:1)
<dom>     at e.exports.Beautifier.beautify (require.js:1)
<dom>     at e.exports (require.js:1)
<dom>     at Object.html_beautify (require.js:1)
<dom>     at Object.toString (require.js:68)
<dom>     at Object.show (require.js:68)
<dom>     at Object.e.show (require.js:68)
<dom>     at require.js:1
<dom>     at Object.execCb (require.js:1)
<tripu> same here, dom
```

**Comment by vr000m** [comment]
merge or not merge?

**Comment by marcoscaceres** [comment]
Will try to take a look in next 24 hours. Thanks for reporting the issue!

File: .travis.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ addons:
       - g++-4.8
 before_install:
   - export CXX="g++-4.8" CC="gcc-4.8"
-  - nvm install 6
+  - nvm install 8
```
Comment (marcoscaceres) [inline .travis.yml]: ReSpec will track Node lts starting 31st of October, so please be sure to change this to lts to avoid issues in the future.

Comment (alvestrand) [reply .travis.yml]: At the moment LTS seems to be 6.11.4 - I assume ReSpec won't track it until it moves to 8?

Comment (marcoscaceres) [reply .travis.yml]: Yep, waiting till 8. Just 1 week away.

**Comment by vivienlacourba** [comment]
closing this PR which got merged to master manually.

**Pull request closed without merging**